dbeam-core/src/test/java/com/spotify/dbeam/options/InputAvroSchemaTest.java

```diff
@@ -0,0 +1,190 @@
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.sql.SQLException;
+import java.util.Optional;
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaParseException;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class InputAvroSchemaTest {
+
+  private static File avroSchemaFile;
+  private static Path avroSchemaFilePath;
+  private static String avroSchemaFilePathStr;
+
+  @BeforeClass
+  public static void beforeAll() throws IOException {
+    String jsonSchema =
+        "{\n"
+        + "  \"name\": \"Show\",\n"
+        + "  \"doc\": \"Record description\",\n"
+        + "  \"namespace\": \"v2\",\n"
+        + "  \"type\": \"record\",\n"
+        + "  \"fields\": [\n"
+        + "    {\n"
+        + "      \"name\": \"field1\",\n"
+        + "      \"type\": \"string\",\n"
+        + "      \"doc\": \"Field1 description\"\n"
+        + "    },\n"
+        + "    {\n"
+        + "      \"name\": \"field2\",\n"
+        + "      \"type\": \"string\",\n"
+        + "      \"doc\": \"Field2 description\"\n"
+        + "    }\n"
+        + "  ]\n"
+        + "}\n";
+
+    avroSchemaFile = createTestAvroSchemaFile(jsonSchema);
+    avroSchemaFilePath = avroSchemaFile.toPath();
+    avroSchemaFilePathStr = avroSchemaFilePath.toString();
+  }
+
+  private static File createTestAvroSchemaFile(final String jsonSchema) throws IOException {
+    File newTempFile = File.createTempFile("dataType1", ".avsc");
+    newTempFile.deleteOnExit();
+    Files.write(
+        newTempFile.toPath(),
+        jsonSchema.getBytes(StandardCharsets.UTF_8),
+        StandardOpenOption.CREATE);
+    return newTempFile;
+  }
+
+  @AfterClass
+  public static void afterAll() throws IOException {
+    Files.delete(avroSchemaFile.toPath());
+  }
+
+  @Test
+  public void checkReadAvroSchema() throws IOException {
+
+    JdbcExportPipelineOptions options =
+        PipelineOptionsFactory.create().as(JdbcExportPipelineOptions.class);
+    options.setAvroSchemaFilePath(avroSchemaFilePathStr);
+
+    Assert.assertEquals(avroSchemaFilePathStr, options.getAvroSchemaFilePath());
+
+    final Schema inputSchema = BeamJdbcAvroSchema.parseInputAvroSchemaFile(avroSchemaFilePathStr);
+
+    Assert.assertEquals("Record description", inputSchema.getDoc());
+    Assert.assertEquals("Field1 description", inputSchema.getField("field1").doc());
+    Assert.assertEquals("Field2 description", inputSchema.getField("field2").doc());
+  }
+
+  @Test
+  public void checkReadAvroSchemaWithEmptyParameter() throws IOException {
+
+    String path = "";
+    JdbcExportPipelineOptions options =
+        PipelineOptionsFactory.create().as(JdbcExportPipelineOptions.class);
+    options.setAvroSchemaFilePath(path);
```

Comment:
Do we need to handle empty string path? DBeam takes other paths as parameters and I don't recall handling empty strings. I think it have been fine so far.